### PR TITLE
Example notebooks and documentation

### PR DESCRIPTION
**Pull request by esgomezm**

### CODE REVIEW

**Comment by esgomezm** [comment]
@FynnBe  here is the new notebook and the doc updated but all the test to the PR fail and I really don't know why

**Comment by FynnBe** [comment]
> @FynnBe here is the new notebook and the doc updated but all the test to the PR fail and I really don't know why

test failiure was unrelated and hopefully fixed now

File: .DS_Store
Comment (FynnBe) [inline .DS_Store:1]: this file shouldn't be here?

**Review by FynnBe — CHANGES_REQUESTED** [review]
couldn't review in detail now... just one minor file too many ;-)

**Pull request merged**